ProGetAutomation/Functions/Get-ProGetAsset.ps1

```diff
@@ -2,20 +2,22 @@ function Get-ProGetAsset
 {
     <#
         .SYNOPSIS
-        Used to get assets to the ProGet asset manager. 
+        Gets assets from ProGet. 

```

Comment:
Change synopsis to: `Gets metadata about items in an asset directory.`

Reply:
Done